public/app/features/admin/UserAdminPage.tsx

```diff
@@ -31,37 +40,29 @@ import { contextSrv } from 'app/core/core';
 
 interface Props extends GrafanaRouteComponentProps<{ id: string }> {
   navModel: NavModel;
-  user: UserDTO;
+  user?: UserDTO;
   orgs: UserOrg[];
   sessions: UserSession[];
-  ldapSyncInfo: SyncInfo;
+  ldapSyncInfo?: SyncInfo;
   isLoading: boolean;
-  error: UserAdminError;
-
-  loadAdminUserPage: typeof loadAdminUserPage;
-  revokeSession: typeof revokeSession;
-  revokeAllSessions: typeof revokeAllSessions;
-  updateUser: typeof updateUser;
-  setUserPassword: typeof setUserPassword;
-  disableUser: typeof disableUser;
-  enableUser: typeof enableUser;
-  deleteUser: typeof deleteUser;
-  updateUserPermissions: typeof updateUserPermissions;
-  addOrgUser: typeof addOrgUser;
-  updateOrgUserRole: typeof updateOrgUserRole;
-  deleteOrgUser: typeof deleteOrgUser;
-  syncLdapUser: typeof syncLdapUser;
-}
-
-interface State {
-  // isLoading: boolean;
+  error?: UserAdminError;
+
+  loadAdminUserPage: Dethunked<typeof loadAdminUserPage>;
+  revokeSession: Dethunked<typeof revokeSession>;
+  revokeAllSessions: Dethunked<typeof revokeAllSessions>;
+  updateUser: Dethunked<typeof updateUser>;
+  setUserPassword: Dethunked<typeof setUserPassword>;
+  disableUser: Dethunked<typeof disableUser>;
+  enableUser: Dethunked<typeof enableUser>;
+  deleteUser: Dethunked<typeof deleteUser>;
+  updateUserPermissions: Dethunked<typeof updateUserPermissions>;
+  addOrgUser: Dethunked<typeof addOrgUser>;
+  updateOrgUserRole: Dethunked<typeof updateOrgUserRole>;
+  deleteOrgUser: Dethunked<typeof deleteOrgUser>;
+  syncLdapUser: Dethunked<typeof syncLdapUser>;
```

Comment:
Could we do [this](https://github.com/grafana/grafana/blob/main/contribute/style-guides/redux.md#typing-of-connected-props) here instead of explicitly typing these props and introducing a helper type?